.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+on:
+  push:
+    tags:
+      - v[0-9]+\.[0-9]+\.[0-9]+
```

Comment:
I think linting can be run on all pushes, or at least merge requests. wdyt?

Reply:
Yes makes sense. I just didn't want to clutter GitHub actions but since this project is relatively small it makes sense to lint more often.